package.json

```diff
@@ -10,7 +10,7 @@
     "publish:packages": "lerna publish",
     "start": "turbo start",
     "test": "turbo test",
-    "test:cypress:dev": "turbo test:cypress:dev",
+    "test:cypress:open": "turbo test:cypress:open",
```

Comment:
Why are you changing this script?

Reply:
Adrian told me to 

- because when i ran the cypress test, tha terminal just freze

Reply:
If you want to use the command `test:cypress:open` then you should add a new script rather than modify the existing one as this will break the current workflow.

You could just add `"test:cypress:open": "turbo test:cypress:open",` next to the `test:cypress:dev` script.